src/styles/blocks/fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: Avenir;
```

Comment:
Using the same `font-family` name 'Avenir' for different font files without specifying different `font-weight` or `font-style` properties can lead to conflicts. Consider renaming this to something more specific or ensuring it is used consistently.